tools/isobuild/compiler-plugin.js

```diff
@@ -776,8 +776,14 @@ export class PackageSourceBatch {
 
       map.forEach((info, name) => {
         if (! name) return;
+
+        let mainModule = _.find(info.files, file => file.mainModule);
+        mainModule = mainModule ?
+          `meteor/${name}/${mainModule.targetPath}` : false;
```

Comment:
Thanks for the quick look!  `installPath` isn't available yet, since currently the code runs just before the import-scanner.  Would you prefer to move it to after?  The import scanner takes the existing map as input, so not sure if any side effects here, or on mutating the generated scannerMap.

I did a quick test and started getting "Can't find npm module" errors for e.g. 'meteor/underscore', but I might have rushed too much.  Also `installPath` is like `node_modules/meteor/...` so would need to either strip the `node_modules/` or prefix with `/`.